nominal/sdk.py

```diff
@@ -382,6 +497,7 @@ def create_run(
         end: datetime | IntegralNanosecondsUTC,
         *,
         datasets: Mapping[str, str] | None = None,
+        log_sets: Mapping[str, str] | None = None,
```

Comment:
I removed the dataset argument in the new functional API PR, just makes the SDK simpler to use and combining this API call isn't really solving an important headache (i.e. it's okay to do these operations step-by-step)